Dockerfile

```diff
@@ -60,22 +60,16 @@ RUN git clone --depth=1 --branch rpm-lockfiles https://github.com/redhat-exd-reb
 # Replace package.json version for this build
 RUN sed -i "s/0.0.0-semantic-release/${RENOVATE_VERSION}/g" package.json
 
-# Install project dependencies
-RUN pnpm install
-
-# Build Renovate
-RUN pnpm build
-
-# Install executables into the bin dir
-RUN npm install --prefix /home/renovate .
+# Install project dependencies, build and install Renovate
+RUN pnpm install && pnpm build && npm install --prefix /home/renovate . && pnpm store prune && npm cache clean --force
 
 WORKDIR /home/renovate/rpm-lockfile-prototype
 
 # Clone and install the rpm-lockfile-prototype
 # We must pass --no-dependencies, otherwise it would try to
 # fetch dnf from PyPI, which is just a dummy package
 RUN git clone --depth=1 --branch v0.1.0-alpha.7 https://github.com/konflux-ci/rpm-lockfile-prototype.git .
-RUN pip3 install --user jsonschema PyYaml productmd requests
-RUN pip3 install --user --no-dependencies .
+RUN pip3 install --user jsonschema PyYaml productmd requests && pip3 install --user --no-dependencies . && pip3 cache purge
 
+RUN chmod -R 755 /home/renovate
```

Comment:
Could this be moved just after creating the home dir? It would speed up the build.

Reply:
I'm afraid it could cause some issues, the dir/files created later (during package installations) after this `chmod` step won't have permission of `755`, but I will test that.

Reply:
Yeah it looks like most files have 0644 and dirs 0755

Reply:
I verified `chmod` early does work, though that was tested with `rpm-lockfile-prototype` installed globally (because installing to user dir was causing another error of `rpm_lockfile` module not found and PYTHONPATH doesn't help).
The PR has been updated.